corehq/apps/sso/tasks.py

```diff
@@ -141,6 +141,12 @@ def auto_deactivate_removed_sso_users():
             notify_exception(None, f"Failed to get members of the IdP. {str(e)}")
             send_deactivation_skipped_email(idp=idp, failure_reason=MSGraphIssue.HTTP_ERROR)
             continue
+        except Exception as e:
+            notify_exception(None, f"Failed to get members of the IdP. {str(e)}")
+            send_deactivation_skipped_email(idp=idp, failure_reason=MSGraphIssue.VERIFICATION_ERROR,
+                                            error="verification error",
+                                            error_description={str(e)})
```

Comment:
We should avoid sending the raw error to the user, in the chance it contains sensitive information we didn't expect. `notify_exception` is fine and necessary, but for this we should say "unknown issue: please contact support"

Reply:
thanks!

Reply:
Hi Biyeun, changed in 8871badc4af1022a6b2cfa5dccbc78483274bec9